kin-sdk-core/src/androidTest/java/kin/sdk/core/KinClientTest.java

```diff
@@ -19,15 +19,11 @@
 
     private static final String PASSPHRASE = "testPassphrase";
 
-    @Test
-    public void testWrongServiceProvider() {
+    @Test(expected = EthereumClientException.class)
```

Comment:
I want to make sure the message is a provider.

Reply:
fixed, now considering the message also